src/crystal/system/addrinfo.cr

```diff
@@ -0,0 +1,39 @@
+module Crystal::System::Addrinfo
+  # alias Handle
+
+  # protected def initialize(addrinfo : Handle)
+
+  # def system_ip_address : ::Socket::IPAddress
+
+  # only used by `#system_ip_address`?
+  # def to_unsafe
```

Comment:
**question:** Why is `to_unsafe` part of the interface? It seems to be used only internally in the implementations.

Reply:
It has to be somewhere, I don't think we could outright remove it even if we expand the internal use of it.

Reply:
No, I mean we can keep the actual `def to_unsafe` but drop the commented pseudo-interface definition here. That's only intended for methods that are called externally.